ballerina/endpoint.bal

```diff
@@ -23,16 +23,26 @@ import ballerina/jballerina.java;
 public isolated client class Client {
     private ConnectionConfig clientEndpointConfig = {};
```

Comment:
Is this field used anywhere other than the `init` function ? If not shall we remove this as well :thinking:

Reply:
+1 removed with https://github.com/ballerina-platform/module-ballerinax-redis/pull/197/commits/6d826c80794537dc41bc07fc1bf660473cc4c8c5